### ISSUE TITLE
"Espanol" button on patient experience should be marked as Spanish-language

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#3975]
## Area or Component
Patient experience. (self-registration and results)

## Quick Summary
The button to switch to the Spanish version of the patient experience is not itself styled as Spanish

## Accessibility Violation
The change in language for a portion of content is not coded. https://axe.deque.com/rule-help/en/lang-change-not-marked

In practice, this will make the text unreadable for screenreader users (imagine pronouncing "Espanol" with an American accent - E- span-oll or similar)

## Action Requested
Style the button to be `lang=es`

## Source Code (optional)
Probably the [LanguageToggler](https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/patientApp/LanguageToggler.tsx#L4-L4)

## Original Deque Issue:
lang-change-not-marked | The change in language for a portion of content is not coded. | The "Espanol" button is not labelled as Spanish language. In practice, this means that screenreaders will read with an English accent. We should also check to ensure the Spanish translations are language-labelled correctly | Change in language is not marked | https://axe.deque.com/rule-help/en/lang-change-not-marked | serious | Yes | No | Structure | ["body > div#root:nth-of-type(1) > div.bg-base-lightest.minh-viewport:nth-of-type(1) > div.bg-white:nth-of-type(1) > header.header.border-bottom.border-base-lighter:nth-of-type(1) > div.display-flex.flex-align-center.maxw-tablet.grid-container.patient-header:nth-of-type(1) > div.display-flex.flex-align-end:nth-of-type(2) > button.usa-button.usa-button--unstyled:nth-of-type(1)"] | The change in language for a portion of content is not coded. |   | wcag2aa,wcag312a | rnv7@cdc.gov | Patient Self-Registration - Form | https://stg.simplereport.gov/app/register/9C7BB |   | 2022-07-13T22:16:55.472Z
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
